android/app/src/main/java/smart/delirover/app/MailmanTraveledDistanceActivity.java

```diff
@@ -0,0 +1,234 @@
+package smart.delirover.app;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.Toast;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.delirover.app.R;
+
+import org.eclipse.paho.client.mqttv3.IMqttActionListener;
+import org.eclipse.paho.client.mqttv3.IMqttDeliveryToken;
+import org.eclipse.paho.client.mqttv3.IMqttToken;
+import org.eclipse.paho.client.mqttv3.MqttCallback;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+//MQTT connection part of the code is built on https://github.com/DIT112-V21/smartcar-mqtt-controller.git
+
+public class MailmanTraveledDistanceActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private static final String TAG = "SmartcarMqttController";
+    private static final String EXTERNAL_MQTT_BROKER = "aerostun.dev";
+    private static final String LOCALHOST = "10.0.2.2";
+    //private static final String TA_SERVER = "aerostun.dev";
+    private static final String MQTT_SERVER = "tcp://" + EXTERNAL_MQTT_BROKER + ":1883";
+    private static final String THROTTLE_CONTROL = "/smartcar/control/throttle";
+    private static final String STEERING_CONTROL = "/smartcar/control/steering";
+
+    private static final int MOVEMENT_SPEED = 70;
+    private static final int IDLE_SPEED = 0;
+    private static final int STRAIGHT_ANGLE = 0;
+    private static final int STEERING_ANGLE = 50;
+    private static final int QOS = 1;
+    private static final int IMAGE_WIDTH = 320;
+    private static final int IMAGE_HEIGHT = 240;
+
+    Context context;
+
+    
+    // I'm unfamilar with reusing the MqttClient, so I have no choice but to keep the code as it is.

```

Comment:
You already copy pasted the MqttClient, so it's difficult to see how "you have no choice" to reuse it. Please reuse it and don't respond with code comments to pull request review comments.